### ISSUE TITLE
selectedIndex not working for dynamically loaded data and double onChange events

### ISSUE THREAD

**Issue by pambrose** [rjaros/kvision#463]
Hi Robert,

First, a note of sincere appreciation for all your efforts with kvision. It is a fantastic framework that is a pleasure to use. 

I created a demo repo for my two issues: https://github.com/pambrose/kvision-issues

If you run the app, you will see that changing the *selectedIndex* value has no effect on a select list if the data was loaded dynamically. No matter what value you use on [line 51](https://github.com/pambrose/kvision-issues/blob/b03daad71cc2dbd3b79d77e6fdca56b6efcc12ef/src/main/kotlin/com/example/doubleevent/App.kt#L51) of App.kt, you always get the first item selected. The *selectedIndex* value works fine on static data.

The second issue can be seen by editing the *Date Value* column and then editing the *Text Value* column. You will see that the date column generates two change events, while the text column generates a single event. 

Any thoughts on these matters would be appreciated.

Cheers,
Paul

**Comment by rjaros** [comment]
Hi,
Unfortunately I can't reproduce the first issue in your example. What browser do you use?

**Comment by rjaros** [comment]
As for the second issue, it's a bug I've just fixed. Unfortunately you will still get two "change" events if you change the date from the keybord, because additional change event is dispatched directly by the text field. If it is a problem you can use dedicated "change.td" event:
```kotlin
onEvent {
    event("change.td") {
        console.log("date changed")
    }
}
```

**Comment by pambrose** [comment]
I saw the first issue in Safari. But strangely, it is now working for me as well, which is really weird. So please ignore the first issue. Thanks for the fix on the double events issue.

**Comment by rjaros** [comment]
Select component was fixed in the last release (see #459) . Perhaps you were using an older version.

**Comment by rjaros** [comment]
Fixed in 6.2.0
